### PR TITLE
Multiple improvements

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Lukas, thank you for the wonderfully useful tool and for all your help using it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,22 +19,22 @@ CopyQ is advanced clipboard manager with editing and scripting features.
 
 ## Overview
 
-CopyQ monitors system clipboard and saves its content in customized tabs.
-Saved clipboard can be later copied and pasted directly into any application.
+* CopyQ monitors the system clipboard and saves its content in customized tabs.
```
Comment (hluk) [inline README.md]:
```suggestion
* CopyQ monitors the system clipboard and saves its content in customized tabs.
```

File: README.md [hunk 2]
```diff
@@ -75,15 +75,15 @@ choco install copyq
...
 ```bash
 brew install --cask copyq
 ```
 
 ### Debian 10+, Ubuntu 18.04+, and their derivatives
 
-Install `copyq` package.
+Install `copyq` package:
 
 `copyq-plugins` is highly recommended. `copyq-doc` available.
```
Comment (hluk) [inline README.md]: The colon is not correct here. Probably best to simplify it as:

```suggestion
Install `copyq` and `copyq-plugins` packages.
```

**Review by hluk — COMMENTED** [review]
Thanks for proofreading and fixing the README file!

**Comment by Gitoffthelawn** [comment]
You're most welcome.  And thank *you*.

**Pull request merged**